bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,16 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.0
+************
+
+* Removed the ``nrf_inbuilt_key`` API.
+  From now on, the application is responsible for handling keys.
```

Comment:
Sorry, another update after talking to Glenn. ;)

```
* Removed the ``nrf_inbuilt_key`` API.
  From now on, the application is responsible for provisioning keys using the AT command **%CMNG**.
```

Reply:
Updated.